datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java

```diff
@@ -3779,10 +3779,10 @@ public void testThroughputViolatingTopicsHandlingForMultipleDatastreams() throws
     Coordinator coordinator = createCoordinator(_zkConnectionString, testCluster, properties);
     String numThroughputViolatingTopicsMetricForFirstDatastream =
         String.format("%s.%s.%s", Coordinator.class.getSimpleName(),
```

Comment:
nit: consider changing this to a function. This format is used in multiple places.

Reply:
done